### PR TITLE
381 Set up Travis CI

### PR DESCRIPTION
**Pull request by liamtoozer**
# Motivation and Context

We need to use start using TravisCI to run unit tests on our PRs. 

# What has changed

**Note: Integration tests and Vulture linting are not being checked by TravisCI as they need further work**
* Add Travis build config for running vulnerability check, flake8 and unit tests
* Fix existing Flake8 linting failures
* Remove unused `pyjwt` dependency as this was failing vulnerability check with pipenv 
* Remove redundant cloudbuild config

# How to test?

* Run a `pipenv clean` to remove the redundant package
* Check the commands in Travis work locally

# Links

https://trello.com/c/kPnanCZg
# Screenshots (if appropriate):

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,7 +22,7 @@ serve:
 run:
 	pipenv run inv run
 
-test: flake8 unittests
```
Comment (ryangrundy7) [inline Makefile:25]: The flake8 issues don't seem that bad. I think it's just worth fixing them on this while you're here

File: cloudbuild.yaml [hunk 2]
```diff
@@ -1,27 +0,0 @@
-steps:
-  - name: python:3.10.4-buster
-    id: Python Env Setup And App Tests
-    entrypoint: bash
```
Comment (ryangrundy7) [inline cloudbuild.yaml:4]: As part of this we should be disabling/deleting the trigger that's in the gcp project it's running in

File: Makefile (hunk 1 above)
Comment (liamtoozer) [reply Makefile:25]: It was mostly the vulture failures which needed a bit more investigation, so wondered if we could just fix them altogether

Comment (ryangrundy7) [reply Makefile:25]: Christ that's a lot of warnings from vulture. I was just thinking it'd be good to have flake8 running always. Defo need a card on the vulture stuff

**Review by ryangrundy7 — APPROVED** [review]

**Review by LukeLoze1 — APPROVED** [review]

**Pull request merged**